### PR TITLE
Prepare release 1.10.0

### PR DESCRIPTION
**Pull request by corona10**

### CODE REVIEW

File: pyperformance/__init__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import sys
 
 
-VERSION = (1, 0, 9)
+VERSION = (1, 10, 0)
```
Comment (diegorusso) [inline pyperformance/__init__.py:5]: Just out of curiosity. What made you jump from 0 to 10 in the minor number?

Comment (corona10) [reply pyperformance/__init__.py:5]: woo.. it was my mistake.

Comment (diegorusso) [reply pyperformance/__init__.py:5]: ok, so it should have been 1.0.10 then :) What are you going to do? Do you leave as it is or will you withdraw the release and re-release it with the fixed version?

Comment (corona10) [reply pyperformance/__init__.py:5]: Unfortunately, it was already released so we decided to keep the current version.
Sorry for the confusion :(

Comment (diegorusso) [reply pyperformance/__init__.py:5]: I'm fine with that! thanks for the update!

**Pull request merged**